app/services/parsers/pdf.rb

```diff
@@ -6,34 +6,13 @@ def initialize(document)
       @document = document
     end
 
-    def pages
-      reader.pages.size
-    end
-
-    def page(index)
-      reader.page(index).text
-    end
-
     def text
-      @text ||= begin
-        text = ""
-        reader.pages.each do |page|
-          text << page.text
-        end
-
-        text
-      end
-    end
-
-    private
-
-    def reader
-      @reader ||= begin
-        io = StringIO.new
-        io.puts(@document.contents)
-
-        PDF::Reader.new(io)
-      end
+      # NOTE: Using -raw opt causes text to be broken up a lot; but not using raw
+      #       may cause tables to be "pretty" in text which may not be ideal for chunking.
+      #       Not specifying works best for rotated pages, so doing that for now
+      cmd = 'pdftotext - -'
+      txt, = Open3.capture2(cmd, stdin_data: @document.contents, binmode: true)
```

Comment:
What do you think about capturing stderr and logging it if present?

Reply:
Done.